examples/ui/ui.rs

```diff
@@ -49,7 +49,7 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
                         .spawn(NodeBundle {
                             style: Style {
                                 size: Size::new(Val::Percent(100.0), Val::Percent(100.0)),
-                                align_items: AlignItems::FlexEnd,
+                                align_items: AlignItems::FlexStart,
```

Comment:
We might be able to remove this, it will probably look the same.

Reply:
This one looks the same without the align_self. Removed!